src/components/CropDetailsModal/CropDetailsModal.js

```diff
@@ -59,15 +59,7 @@ const CropDetailsModal = ({ crop, setModalOpen, modalOpen }) => {
   };
 
   const print = () => {
-    if (state.consent === true) {
```

Comment:
we still want this. it's a google analytics event saying the user is exporting a pdf